v2/pages/userInstanceQuestionExam/userInstanceQuestionExam.ejs

```diff
@@ -47,6 +47,34 @@
               </div>
             </div>
             <% } %>
+            
+            <% if (showAllSubmissions) { %>
+            <div class="panel-group">
+                <% for (sNum = 0; sNum < allSubmissions.length; sNum++) { 
+                thisSubmission = allSubmissions[sNum]; 
+                if (thisSubmission.correct === null) {
+                    panelType = "panel-info"; 
+                    submissionLabel = "<span class='label label-primary'>saved, not graded</span>"; 
+                }
+                else if (thisSubmission.correct) {
+                    panelType = "panel-success";
```

Comment:
I feel like having different panelTypes introduces too much visual noise in the display. I suggest sticking with just a single panelType (maybe `panel-info`?) and indicating correctness just with the label.